package.json

```diff
@@ -5,65 +5,61 @@
   "angular-cli": {},
   "scripts": {
     "start": "ng serve",
-    "lint": "tslint --fix \"src/**/*.ts\"",
+    "lint": "tslint \"src/**/*.ts\"",
+    "lint:fix": "tslint --fix \"src/**/*.ts\"",
     "test": "ng test",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
     "sw": "sw-precache --root=dist --config=sw-precache-config.js"
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^2.4.1",
-    "@angular/compiler": "^2.4.1",
-    "@angular/core": "^2.4.1",
-    "@angular/forms": "^2.4.1",
-    "@angular/http": "^2.4.1",
-    "@angular/platform-browser": "^2.4.1",
-    "@angular/platform-browser-dynamic": "^2.4.1",
-    "@angular/router": "^3.4.1",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.15",
-    "@ngrx/core": "^1.2.0",
-    "@ngrx/effects": "^2.0.0",
-    "@ngrx/router-store": "^1.2.5",
-    "@ngrx/store": "^2.2.1",
-    "@ngrx/store-devtools": "^3.2.2",
-    "bootstrap": "next",
+    "@angular/common": "2.4.3",
+    "@angular/compiler": "2.4.3",
+    "@angular/core": "2.4.3",
+    "@angular/forms": "2.4.3",
+    "@angular/http": "2.4.3",
+    "@angular/platform-browser": "2.4.3",
+    "@angular/platform-browser-dynamic": "2.4.3",
+    "@angular/router": "^3.4.3",
+    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.18",
+    "bootstrap": "^3.3.7",
     "core-js": "^2.4.1",
     "immutable": "^3.8.1",
     "jquery": "^3.1.1",
-    "lodash": "^4.17.2",
-    "ng2-restangular": "^0.1.23",
+    "lodash": "^4.17.4",
+    "ng2-restangular": "^0.1.27",
     "patternfly": "^3.17.0",
     "pluralize": "^3.1.0",
     "reselect": "^2.5.4",
-    "rxjs": "^5.0.1",
+    "rxjs": "^5.0.3",
     "tether": "^1.4.0",
     "ts-helpers": "^1.1.2",
-    "zone.js": "^0.7.4"
+    "zone.js": "^0.7.5"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "^2.4.1",
-    "@types/jasmine": "^2.5.38",
-    "@types/lodash": "^4.14.44",
-    "@types/node": "^6.0.52",
+    "@angular/compiler-cli": "2.4.3",
+    "@types/jasmine": "^2.5.40",
```

Comment:
Now that we use yarn I don't think we need to bother that. Upgrading dependencies should be a considered action using `yarn upgrade ...` which will update `yarn.lock` accordingly. Normal dev should just be `yarn` (or `yarn install`) to install dependencies as defined in lock file.